pyHS100/pyHS100.py

```diff
@@ -103,7 +103,7 @@ def _query_helper(self, target, cmd, arg={}):
                 request={target: {cmd: arg}}
             )
         except Exception as ex:
-            raise SmartPlugException(ex)
+            raise SmartPlugException(ex) from ex
```

Comment:
I've tried adding `from future.utils import raise_from` on my local copy, but now I get import errors about future.utils. I think I need to add it as a dependency, but I'm not sure how to do that and maintain python2/3 compatibility.